### PR TITLE
Show a useful icon in SQL tabs

### PR DESCRIPTION
**Pull request by mgrojo**
The icon shows whether the tab is linked to a project file (tab icon) or
to a file in disk (document_open icon).

This has a lateral effect of fixing this, since the tab always has an icon
and the height of the tab bar never changes:

https://github.com/sqlitebrowser/sqlitebrowser/issues/2073#issuecomment-570900792

I had this change in my local repository and don't want to merge it so close to the release without some feedback. The code changes are minimal, but I'm not sure about whether we want icons there, or what icons should be used. For #2073 there is probably other solution, but I was thinking some time ago that it would be nice to have a visual way to know whether the tab is saved in a file or in the project file. At some time we might also want to have SQL files saved in the project file as a reference to an external file, and not as a saved text embedded in the project.

### CODE REVIEW

File: src/MainWindow.cpp [hunk 1]
```diff
@@ -343,6 +343,8 @@ void MainWindow::init()
     });
     ui->tabSqlAreas->setCornerWidget(buttonCloseSqlTab);
 
+    //QIcon execIcon(":icons/hourglass");
+    ui->tabSqlAreas->setMinimumSize(ui->tabSqlAreas->iconSize());
```
Comment (MKleusberg) [inline src/MainWindow.cpp]: Is this still needed?

File: src/MainWindow.cpp [hunk 2]
```diff
@@ -1200,6 +1207,7 @@ void MainWindow::executeQuery()
     // NOTE It's a bit hack-ish but we don't use this icon just as a signal to the user but also check for it in various places to check whether a
     // specific SQL tab is currently running a query or not.
```
Comment (MKleusberg) [inline src/MainWindow.cpp]: Can you remove that comment, not that it is outdated? Or better yet, update it to explain that the tab data value means :smile:

File: src/MainWindow.cpp (hunk 1 above)
Comment (mgrojo) [reply src/MainWindow.cpp]: No, sorry. A test leftover.

File: src/MainWindow.cpp (hunk 2 above)
Comment (mgrojo) [reply src/MainWindow.cpp]: Of course. I'll update.

**Comment by mgrojo** [comment]
@MKleusberg I've fixed the observations and I propose another icon, which we already had but without use.

**Comment by mgrojo** [comment]
I've merged this to master, since the observations had been fixed and now we have a 3.12.x version and the decision to merge it to that release can be taken later.

**Comment by MKleusberg** [comment]
Thanks, @mgrojo! And sorry for the delay on my side :innocent:

**Comment by mgrojo** [comment]
Don't worry! You had already reviewed it. Should we consider it as candidate for 3.12.x? I fear spartan users who never seem to have enough space, though :wink:

**Comment by MKleusberg** [comment]
Yeah, let's include it in 3.12.x because besides being more useful it somewhat fixes the multithreading code we introduce in that release :smile:

**Pull request merged**